extensions/cornerstone-dicom-sr/src/getSopClassHandlerModule.ts

```diff
@@ -132,6 +123,12 @@ function _getDisplaySetsFromSeries(
   return [displaySet];
 }
 
+/**
+ * Loads the display set with the given services and extension manager.
+ * @param {Object} displaySet - The display set to load.
```

Comment:
You don't need to include the type prefix if you area defining the types within the function call itself.

Reply:
Sounds good, I'll update it.